src/main/java/Bebe.java

```diff
@@ -0,0 +1,330 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Bebe {
+
+    private static final String FILE_PATH = "./data/bebe.txt";
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        loadTasksFromFile();
+        welcomeMessage();
+        runChatbot();
+    }
+
+    /**
+     * Prints a welcome message when the chatbot starts.
+     */
+    private static void welcomeMessage() {
+        System.out.println("Hello! I'm Bebe");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Runs the main loop of the chatbot to handle user input.
+     */
+    private static void runChatbot() {
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+
+        while (true) {
+            userInput = scanner.nextLine().trim();
+            String[] words = userInput.split(" ", 2);
+
+            try {
+                processUserInput(words);
+            } catch (BebeException e) {
+                System.out.println("OOPS!!! " + e.getMessage());
+            } catch (Exception e) {
+                System.out.println("Unexpected error: " + e.getMessage());
+            }
+        }
+    }
+
+    /**
+     * Processes the user's input and calls the appropriate methods.
+     */
+    private static void processUserInput(String[] words) throws BebeException {
+        switch (words[0].toLowerCase()) {
```

Comment:
Maybe refactoring `words[0]` to indicate that it is a command would make it clearer?